niceman/resource/session.py

```diff
@@ -563,6 +573,12 @@ def mkdir(self, path, parents=False):
         if not self.isdir(path):
             raise CommandError(cmd='mkdir', msg="Failed to create directory")
 
+    def mktmpdir(self):
+        path, err = self.execute_command(["mktemp", "-d"])
+        if err:
+            raise SessionRuntimeError("mktmpdir had error output: %s" % err)
```

Comment:
Do you know of a case where `mktemp -d` has an exit code of 0 and stderr?  Otherwise, I think `execute_command` will throw a `CommandError` before this is reached.

Reply:
No, I don't.  I did model this after the "read" command two lines above.  But I'm happy to simplify the code (mkdir is also simpler)